### PR TITLE
Integrating In-Progress a11y updates for Application Info

### PR DESCRIPTION
**Pull request by ltsheu**
The accessibility updates are still in progress for Application Info, however I would like to get everything we have so far onto the integration site, to facilitate testing with the JAWS screen reader. 

Related user stories/chores: #287, #301, #306.  Note: These user stories/chores are still in progress until further testing with JAWS.

**NOTE:** With this PR, **the help buttons and help text will only function on the Application Info page**. The help buttons/text will *not* function on other pages until the new help pattern is applied to those pages (during a11y remediation for those pages).

**NOTE:** A "Skip to Main Content" link has been added with this PR, however it will not function properly yet. This will be remediated with #307.

### CODE REVIEW

File: DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.js [hunk 1]
```diff
@@ -13,6 +13,15 @@ module.exports = function(ngModule) {
     'use strict';
 
     var vm = this;
+    vm.sections = navService.getSections().map(function(element){
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.js]: Can this be replaced with `vm.sections = this.getSections()`?

File: DOL.WHD.Section14c.Web/src/v4/dol-footer.component.ts [hunk 2]
```diff
@@ -10,7 +10,7 @@ import { Component } from '@angular/core';
           <img
             class="brand--img"
             src="images/dol-logo-white.png"
-            alt="DOL Logo"
+            alt="Seal of the United States Department of Labor"
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/v4/dol-footer.component.ts]: This was one of the first lessons I learned about proper accessibility: never alt an image with the word "logo". Lol

File: DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.js (hunk 1 above)
Comment (ltsheu) [reply DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.js]: @mgwalker Let's ask Harun, I believe this was his change. The changes from his associated chore are part of this PR. (I think this was for chore #306) @davoodharun  can you comment? Thanks.

Comment (davoodharun) [reply DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.js]: I removed this.getSections. It was not needed.

**Pull request merged**